File: ruby/1 - Iniciante/1020.rb
n = gets.chomp.to_i
anos = (n / 365)
mes =  (n % 365) / 30
dias = (n % 365) % 30

puts (anos.to_s + " ano(s)")
puts (mes.to_s + " mes(es)")
puts (dias.to_s + " dia(s)")

File: python3/1 - Iniciante/1020.py
class Tempo:
    def __init__(self, n):
        n = int(n)
        self.year = self.calcYear(n)
        self.month = self.calcMonth(n)
        self.days = self.calcDays(n)

    def calcYear(self, n):
        return divmod(n, 365)[0]

    def calcMonth(self, n):
        return divmod(divmod(n, 365)[1], 30)[0]

    def calcDays(self, n):
        return divmod(divmod(n, 365)[1], 30)[1]

    def printThis(self):
        print(self.year, "ano(s)")
        print(self.month, "mes(es)")
        print(self.days, "dia(s)")

tempo = Tempo(int(input()))
tempo.printThis()